### PR TITLE
doc: Fix wrong helm installation guide

### PR DESCRIPTION
**Pull request by samzong**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

/kind documentation

* **What this PR does / why we need it**:

* **Which issue(s) this PR fixes**:

Fix wrong helm installation guide

modified:   content/en/docs/installation.md
modified:   content/zh/docs/installation.md

### CODE REVIEW

**Comment by samzong** [comment]
Is there anything else that needs updating?

File: content/en/docs/installation.md [hunk 1]
```diff
@@ -20,129 +20,72 @@ The easiest way to get started with Volcano is to download the [release](https:/
...
+- Install with Helm
 
 ### Install with YAML files
 
 Install Volcano on an existing Kubernetes cluster. This way is both available for x86_64 and arm64 architecture.
 
-```
-kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
+```shell
+#kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
```
[the remark this answers is not in the document]
Comment (samzong) [reply content/en/docs/installation.md:36]: maybe there is no need to add `#` or `~`. it will be more convenient when using the copy button on the page

**Comment by Monokaix** [comment]
Thanks for your contribution!
/lgtm

**Review by william-wang — APPROVED** [review]
/approve

File: content/en/docs/installation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevin-wangzefeng) [reply content/en/docs/installation.md:36]: I don’t think we need the `#` symbol here. We can add a `#` at the beginning of each echo line to turn them into comments. 
However, for executable commands, we should not comment them out unless we want to disable their execution.

There are a few more shell scripts in the document that have similar issue, please feel free to correct them as well.

[the remark this answers is not in the document]
Comment (Monokaix) [reply content/en/docs/installation.md:36]: That's right, please remove it and keep a clean commit without merge pr so we can make it forward. @samzong

**Pull request closed without merging**